js/index.js

```diff
@@ -36,3 +36,77 @@ for (let i = 0; i < skills.length; i++) {
   // Append the list item element to the unordered list
   skillsList.appendChild(skill);
 }
+
+// Select the message form by name
+const messageForm = document.forms.leave_message;
+
+// Add event listener for form submit
+messageForm.addEventListener('submit', function(event) {
```

Comment:
This would be a good place to use the anonymous function simplified notation:  (event) =>

Reply:
fixed